ubg/src/App.js

```diff
@@ -38,6 +38,9 @@ function App() {
             <Route exact path="/search">
               <Search />
             </Route>
+            <Route exact path="/search/:query">
```

Comment:
I think instead of adding another route you could just remove the exact from /search and I think it should also work.

Reply:
It gives me an error when I delete it

Reply:
Did you delete the exact from the first Search route?

Reply:
the problem is that if I delete it, it would take "3000/search" as a directory for game